poetry/console/commands/config.py

```diff
@@ -130,7 +130,11 @@ def handle(self):
                 )
 
                 for value in values:
-                    self.line(value[1])
+                    if isinstance(value[1], str) and value[1][0] == value[1][-1] == '"':
```

Comment:
Would you like to add ws check in here?

Reply:
sorry, what do you mean by "ws"?

Reply:
White space characters.

Reply:
I cannot get any command such as `cd $(poetry config settings.virtualenvs.path)` to work no matter what I do.

if `poetry config settings.virtualenvs.path` returns the strings with whitepsace escaped, e.g. a setting with the vaule `/some path/dir with space` returns `/some\ path/dir\ with\ space` usring the command inside like `ls $(...)` still doesn't work.

So I don't know how to proceed. I guess I will drop the whole thing.